fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -210,6 +211,22 @@ public static int analyzePartitionTimeToLive(Map<String, String> properties) thr
         return partitionTimeToLive;
     }
 
+    public static int analyzePartitionRefreshNumber(Map<String, String> properties) throws AnalysisException {
+        int partitionRefreshNumber = -1;
+        if (properties != null && properties.containsKey(PROPERTIES_PARTITION_REFRESH_NUMBER)) {
+            try {
+                partitionRefreshNumber = Integer.parseInt(properties.get(PROPERTIES_PARTITION_REFRESH_NUMBER));
+            } catch (NumberFormatException e) {
+                throw new AnalysisException("Partition Refresh Number: " + e.getMessage());
+            }
+            if (partitionRefreshNumber <= 0) {
+                throw new AnalysisException("Partition Refresh Number should larger than 0.");
```

Comment:
how to unset refresh number?
what if the user do not specify the refresh number, what is the default logic?

Reply:
This feature will do in Support alter materialized view properties or later PR